cpp/ql/lib/semmle/code/cpp/rangeanalysis/new/internal/semantic/analysis/RangeAnalysisStage.qll

```diff
@@ -1106,12 +1106,9 @@ module RangeStage<
         b = bRight and origdelta = odRight and reason = rRight and bLeft instanceof SemZeroBound
       )
       or
-      exists(
-        D::Delta dLeft, D::Delta dRight, boolean fbeLeft, boolean fbeRight, D::Delta odLeft,
-        D::Delta odRight, SemReason rLeft, SemReason rRight
-      |
-        boundedSubOperandLeft(e, upper, b, dLeft, fbeLeft, odLeft, rLeft) and
-        boundedSubOperandRight(e, upper, dRight, fbeRight, odRight, rRight) and
+      exists(D::Delta dLeft, D::Delta dRight, boolean fbeLeft, boolean fbeRight |
+        boundedSubOperandLeft(e, upper, b, dLeft, fbeLeft, origdelta, reason) and
+        boundedSubOperandRight(e, upper, dRight, fbeRight, _, _) and
```

Comment:
Might as well remove the last two columns from `boundedSubOperandRight`.

Reply:
Fixed in 20df63f26d75e79b76c22c2152384da501a900ae.